ui/src/components/ExternalTerm.vue

```diff
@@ -49,7 +49,7 @@ export default class extends Vue {
 
   getLabels (pointer: GraphPointer | null): Literal[] {
     if (pointer) {
-      return pointer.out(schema.name, { language: [this.selectedLanguage, '*'] }).terms
+      return pointer.out(schema.name).terms.filter(isLiteral) as Literal[]
```

Comment:
I had to add `as Literal[]` because for some reason, Typescript was ignoring my filter with guard.

Reply:
Apparently typescript cannot figure out the common type for `.terms` return type

> `Term[] | [BlankNode] | ...etc`